src/third-parties/index.ts

```diff
@@ -1,7 +1,7 @@
-import GooglaAnalyticsData from './google-analytics/data.json';
-import GoogleTagManagerData from './google-tag-manager/data.json';
-import GoogleMapsEmbedData from './google-maps-embed/data.json';
-import GoogleYoutubeEmbedData from './youtube-embed/data.json';
+import GooglaAnalyticsData from '../../data/google-analytics.json';
```

Comment:
Seems like a typo from a previous PR squeezed through:

```suggestion
import GoogleAnalyticsData from '../../data/google-analytics.json';
```